sentence_transformers/losses/CoSENTLoss.py

```diff
@@ -0,0 +1,81 @@
+import torch
+from torch import nn, Tensor
+from typing import Iterable, Dict
+from ..SentenceTransformer import SentenceTransformer
+from .. import util
+
+
+class CoSENTLoss(nn.Module):
+    """
+    This class implements CoSENT (Consistent Sentence Embedding via Similarity Ranking) loss.
```

Comment:
Is CoSENT short for Consistent Sentence Embedding via Similarity Ranking? I thought the blogpost/paper just mentioned "Cosine Sentence".

Reply:
I think you are correct, I see now where the author calls it Cosine Sentence. I was going off the name in [this](https://github.com/RingBDStack/CoSENT/blob/main/README.md) repo, but on second inspection I don't see any source for their choice of name.

Reply:
Just fixed the doc string!